cloudbuild.yaml

```diff
@@ -0,0 +1,17 @@
+timeout: 3000s
+steps:
```

Comment:
can you add a comment at the top of the file describing what the workflow is meant to do, which branch it builds and when it gets triggered please.

Reply:
sure